app/controllers/eredienst-mandatenbeheer/new.js

```diff
@@ -48,11 +49,14 @@ export default class EredienstMandatenbeheerNewController extends Controller {
     event.preventDefault();
 
     let { worshipMandatee } = this.model;
-    yield worshipMandatee.save();
-
-    this.router.transitionTo(
-      'eredienst-mandatenbeheer.mandataris.edit',
-      worshipMandatee.id
-    );
+    if (yield isMandaatSelected(worshipMandatee)) {
+      yield worshipMandatee.save();
+      this.router.transitionTo(
+        'eredienst-mandatenbeheer.mandataris.edit',
+        worshipMandatee.id
+      );
+    } else {
+      worshipMandatee.errors.add('mandaat', 'Mandaat is een vereist veld.');
```

Comment:
The first string should be the name of the attribute or relationship (so that would be 'bekleedt' here). It has the benefit that the error message will be cleared if the value changes. Except... It seems that isn't working for relationships at the moment 😄 (which might be a bug in Ember Data). Maybe it's related to the inheritance, not sure.

```suggestion
      worshipMandatee.errors.add('bekleedt', 'Mandaat is een vereist veld.');
```

I would still clear the error message if the user selects a value after the message is shown. For now I would explicitly [remove](https://api.emberjs.com/ember-data/4.7/classes/Errors/methods/remove?anchor=remove) it in the 'setMandate' action.